docs/installing-windows.md

```diff
@@ -14,6 +14,9 @@ nav_order: 11
 {: .warning }
 Do not use Windows 8 or older. Only recent Windows 10 and 11 builds are supported.
 
+{: .warning }
+Do not use unofficial versions of Windows, such as ghost spectre or tiny1X. They are not supported.
```

Comment:
>such as ghost spectre or tiny1X. They are not supported.

Should be 

such as Ghost Specter or tiny10/11. They are not supported and may contain **undetectable** malware and/or security vulnerabilities.

Reply:
Fixed~